pkg/util/ruskmock/server.go

```diff
@@ -30,7 +30,10 @@ import (
 
 var log = logrus.WithField("process", "mock rusk server")
 
-const stateTransitionDelay = 1 * time.Second
+const (
+	stateTransitionDelay = 1 * time.Second
+	stakeGracePeriod     = 2
```

Comment:
To stay in line with the consensus rules, we should probably keep this at `1000`. I guess you put it to `2` for testing purposes?

Reply:
Until we address all stability and performance issues in consensus, we should allow `stakeGracePeriod=2` in mocking service for both test-harness and devnet testing.